googleapis/spanner/v1/spanner.pb.go

```diff
@@ -2359,56 +2357,6 @@ type SpannerServer interface {
 	PartitionRead(context.Context, *PartitionReadRequest) (*PartitionResponse, error)
 }
 
-// UnimplementedSpannerServer can be embedded to have forward compatible implementations.
-type UnimplementedSpannerServer struct {
```

Comment:
It looks like exported types/methods are removed here (and in many other files). Could this be a breaking change to other projects? Do we have some kind of testing that checks for that? (Sorry for the naive questions, I'm new to these reviews...)

Reply:
FWIW looks like many of the things that are being removed were added in the last regen (https://github.com/googleapis/go-genproto/pull/210/ )

Reply:
Hm, good catch. I just assume this was part of the regen process, but now that you mention it, this does look weird. @noahdietz do you have any insight into what's going on?

Reply:
What version of protoc & protoc-gen-go was used to regenerate? The Unimplimented Server stub was added in https://github.com/golang/protobuf/releases/tag/v1.3.2

It's possible that you are on an older version that doesn't have this change, thus removing it

Reply:
Aha, good catch. Was on v1.3.1.

I'm going to abandon this change for now since the change I wanted to pull in was already available and there's no rush to regen at this time.